templates/nodeovc/schema.capnp

```diff
@@ -0,0 +1,76 @@
+@0xd37ff48ad935931f;
+struct Schema {
+	# Description for the VM
+	description @0 :Text;
+
+	# Name of the VM
+	name @1 :Text;
+	
+	# OpenvCloud
```

Comment:
this comment doesn't give any more info. Should explain what that is, is I understand correctly this is the instance name of the openvcloud client used by this service.